components/cdc/src/delegate.rs

```diff
@@ -613,12 +606,33 @@ impl Delegate {
             rows.push(v);
         }
         self.sink_downstream(rows, index, ChangeDataRequestKvApi::TiDb)?;
-
-        self.sink_downstream(raw_rows, index, ChangeDataRequestKvApi::RawKv)?;
+        self.sink_raw_downstream(raw_rows, index)?;
 
         Ok(())
     }
 
+    fn sink_raw_downstream(&mut self, entries: Vec<EventRow>, index: u64) -> Result<()> {
+        if entries.is_empty() {
+            return Ok(());
+        }
+        let max_raw_ts = entries
```

Comment:
The last entry should have the `max_raw_ts`. So we don't need to iterate all entries.

Reply:
done